oxford-metadata.ttl

```diff
@@ -1301,10 +1339,47 @@
 :membrane_potassium_pump_current_conductance
     a :Annotation ;
     a :MembranePumpRelated ;
+    a :ConductanceOrPermeability ;
     rdfs:label "Membrane potassium pump current conductance" ;
     rdfs:comment "Also known as permeability" .
 
 
+# ========================================================================
+# Terms related to state transition rates, useful for parameter fitting
+# ========================================================================
+
+:TransitionRateRelated
+    a :Category ;
+    rdfs:label "Transition rates and related parameters" .
+
+:TransitionRate
+    a :TransitionRateRelated ;
+    rdfs:label "Markov state transition rates" .
```

Comment:
```suggestion
    rdfs:label "Ion channel state transition rate" .
```

Reply:
Yeah gotta be a bit careful with those :D 

"transition rate" is the language from stats markov models, so most appropriate for single channels?

in the chemistry interpretation they're _not_ rates, but "rate constants" or "rate coefficients", and the rate is the thing you get when you multiply that with a state occupancy

But I can't think of something to call them that avoids the word rate

Reply:
My preference would be "Rate coefficient" (not even mentioning the ion channel model)

Reply:
It's a generic form for a voltage-dependent rate coefficient

Reply:
I don't really like "coefficient" because my instinct is to think they are constants not expressions. I've suggested just "Ion channel state transition rate" because it applies to HH models too.

Reply:
I didn't realise "coefficient" sounded like "constant" to some! I've been using coefficient to avoid that connection (chemists literally call them "constants", most of the time, even though they don't treat them as constants https://en.wikipedia.org/wiki/Reaction_rate_constant)

I'm just a bit worried about calling them "transition rates" which is from the probabilistic markov interpretation, when we use all the other things from chemistry interpretation (even the symbol "k")

Reply:
I think this is fine, I don't imagine any (additional) confusion from ion channel modellers calling them this